common/net/bolt9.py

```diff
@@ -0,0 +1,185 @@
+from math import ceil
+
+# feature_name: odd_bit
+known_features = {
+    "option_data_loss_protect": 0,
+    "initial_routing_sync": 2,
+    "option_upfront_shutdown_script": 4,
+    "gossip_queries": 6,
+    "var_onion_optin": 8,
+    "gossip_queries_ex": 10,
+    "option_static_remotekey": 12,
+    "payment_secret": 14,
+    "basic_mpp": 16,
+    "option_support_large_channel": 18,
+    "option_anchor_outputs": 20,
+}
+
+# Reversed map -> odd_bit : feature_name
+known_odd_bits = {v: k for k, v in known_features.items()}
+
+
+def check_feature_name_bit_pair(name, bit):
+    """
+    Checks whether a given name and bit par match for known features.
+
+    Args:
+        name (:obj:`str`): the feature name.
+        bit (:obj:`int`): the bit position.
+
+    Returns:
+        :obj:`bool`: For known features, returns True if the pair match. For unknown features, returns True if the bit
+        is unknown.
+    """
+
+    if name in known_features:
```

Comment:
You can turn the three `known_features` lookups into a single one with this:

```python 
f = known_features.get(name, None)
```
And then check if `f is None` to see if it was in the dict.